akka-docs/rst/security/2017-02-10-java-serialization.rst

```diff
@@ -0,0 +1,61 @@
+Java Serialization, Fixed in Akka 2.4.17
+========================================
+
+Date
+----
+
+10 Feburary 2017
+
+Description of Vulnerability
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An attacker that can connect to an ``ActorSystem`` exposed via Akka Remote over TCP can gain remote code execution 
+capabilities in the context of the JVM process that runs the ActorSystem if:
+
+* ``JavaSerializer`` is enabled (default in Akka 2.4.x)
+* and TLS is disabled *or* TLS is enabled with ``akka.remote.netty.ssl.security.require-mutual-authentication = false``
+  (which is still the default in Akka 2.4.x)
+* regardless of whether ``untrusted`` mode is enabled or not
+
+Java deserialization is `known to be vulnerable <https://community.hpe.com/t5/Security-Research/The-perils-of-Java-deserialization/ba-p/6838995>`_ to attacks when attacker can provide arbitrary types.
+
+Akka Remoting uses Java serialiser as default configuration which makes it vulnerable in its default form. The documentation of how to disable Java serializer was not complete. The documentation of how to enable mutual authentication was missing (only described in reference.conf).
+
+To protect against such attacks the system should be updated to Akka `2.4.17` or later and be configured with 
+:ref:`disabled Java serializer <disable-java-serializer-scala>`. Additional protection can be achieved when running in an 
+untrusted network by enabling :ref:`TLS with mutual authentication <remote-tls-scala>`.
+
+Please subscribe to the `akka-security <https://groups.google.com/forum/#!forum/akka-security>`_ mailing list to be notified promptly about future security issues.
+
+Severity
+~~~~~~~~
+
+The `CVSS <https://en.wikipedia.org/wiki/CVSS>`_ score of this vulnerability is 3.6 (Low), based on vector `AV:A/AC:H/Au:N/C:P/I:P/A:P/E:F/RL:OF/RC:C <https://nvd.nist.gov/cvss.cfm?calculator&version=2&vector=%28AV:A/AC:H/Au:N/C:P/I:P/A:P/E:F/RL:OF/RC:C%29>`_.
```

Comment:
I think it's easies to apply the other patch afterwards (could be done in this PR)